### PR TITLE
proposition of updating readme

### PR DESCRIPTION
**Pull request by LiquidITGuy**
proposition of updating readme file

adding bold for clicable sections
rename enviromental into environment
add the icon to the process
fix link name of the app who does not match with the current README

self check : 
- [X] not concerned :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ Process:
 - Copy the content into the one-click app structure. See this for the reference: [wordpress.json](https://github.com/caprover/one-click-apps/blob/master/public/v2/apps/wordpress.json)
 - Variables are prefixed with `$$cap`
 - Variables can be anywhere in the JSON and they will be replaced by what user enters
-- There are 2 special variables called `$$cap_appname` and `$$cap_root_domain` which exist for all oneclick apps. If your app needs an enviromental variable that needs to point to the URL of the app, you can use `$$cap_appname.$$cap_root_domain` which resolves to something like `myappname.rootdomain.com`
+- There are 2 special variables called `$$cap_appname` and `$$cap_root_domain` which exist for all oneclick apps. If your app needs environment's variables with the URL value of the app, you can use `$$cap_appname.$$cap_root_domain` which resolves something like `myappname.rootdomain.com`
```
Comment (githubsaturn) [inline README.md:9]: minor: doesn't need `'s`

Comment (githubsaturn) [inline README.md:9]: Resolves needs "to"

Comment (LiquidITGuy) [reply README.md]: done

Comment (LiquidITGuy) [reply README.md]: done

**Comment by LiquidITGuy** [comment]
done but " (NB: fields are not required. If validRegex is not set, the field can be set as empty and ignored by the user.)" added line 10 not on preview MR (sorry, dont know why but github desktop making some magic with commit so in local I got a "take into account dev review" commit but not here ¯\_(ツ)_/¯.
I will use another tool next time to be more easy for next pr

**Pull request merged**